libraries/ESP8266WiFi/src/include/ClientContext.h

```diff
@@ -127,8 +127,21 @@ class ClientContext
         }
         _connect_pending = 1;
         _op_start_time = millis();
+        /*LEA*/
+        //Serial.print("\nClientContext::connect (A)");
+        uint32_t    nextPrint = (_op_start_time + 100);
+        while ((millis() < (_op_start_time + _timeout_ms)) &&
+               (state() != ESTABLISHED)) {
+            yield();
+            if (millis() > nextPrint) {
```

Comment:
Same millis issue.  Plus, I'm not sure what the purpose of this as we're not going to be printing anything anyway...

Would suggest going back to the `delay(_timeout_ms);` for this whole section.

Reply:
Same as above, this is just a q&d solution for the connection problems with the existing implementation! Wasn‘t supposed to be committed here, sorry! Just remove if possible!

Reply:
I find this interesting. Does the current implementation have an issue? Does this resolve it?